ui/src/app/kvm/views/KVMDetails/CoreResources/CoreResources.test.tsx

```diff
@@ -23,4 +27,24 @@ describe("CoreResources", () => {
         ?.includes("core-resources--dynamic-layout")
     ).toBe(true);
   });
+
+  it("renders the pinned core section if data is provided", () => {
+    const wrapper = shallow(
+      <CoreResources
+        cores={{ allocated: 1, free: 2 }}
+        pinned={[1]}
+        available={[2]}
+      />
+    );
+
+    expect(wrapper.find("[data-test='pinned-section]").exists()).toBe(true);
```

Comment:
Looks like the selector syntax is broken (missing a `'` after `'pinned-section`)

Reply:
done!